scripts/deploy.sh

```diff
@@ -34,11 +34,74 @@ function upload_assets {
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-no.pdf" "${SITE_HOME}/assets/tldr-book-no.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-pl.pdf" "${SITE_HOME}/assets/tldr-book-pl.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-pt_BR.pdf" "${SITE_HOME}/assets/tldr-book-pt_BR.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-pt_PT.pdf" "${SITE_HOME}/assets/tldr-book-pt_PT.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-ro.pdf" "${SITE_HOME}/assets/tldr-book-ro.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-ru.pdf" "${SITE_HOME}/assets/tldr-book-ru.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-sh.pdf" "${SITE_HOME}/assets/tldr-book-sh.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-sr.pdf" "${SITE_HOME}/assets/tldr-book-sr.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-sv.pdf" "${SITE_HOME}/assets/tldr-book-sv.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-ta.pdf" "${SITE_HOME}/assets/tldr-book-ta.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-th.pdf" "${SITE_HOME}/assets/tldr-book-th.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-tr.pdf" "${SITE_HOME}/assets/tldr-book-tr.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-uk.pdf" "${SITE_HOME}/assets/tldr-book-uk.pdf"
+  cp -f "${TLDRHOME}/scripts/pdf/tldr-pages-uz.pdf" "${SITE_HOME}/assets/tldr-book-uz.pdf"
+
 
   sha256sum \
     "${SITE_HOME}/assets/index.json" \
     "${SITE_HOME}/assets/"*.zip \
     "${SITE_HOME}/assets/tldr-book.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ar.pdf" \
+    "${SITE_HOME}/assets/tldr-book-bn.pdf" \
+    "${SITE_HOME}/assets/tldr-book-bs.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ca.pdf" \
+    "${SITE_HOME}/assets/tldr-book-da.pdf" \
+    "${SITE_HOME}/assets/tldr-book-de.pdf" \
+    "${SITE_HOME}/assets/tldr-book-es.pdf" \
+    "${SITE_HOME}/assets/tldr-book-fa.pdf" \
+    "${SITE_HOME}/assets/tldr-book-fr.pdf" \
+    "${SITE_HOME}/assets/tldr-book-hi.pdf" \
+    "${SITE_HOME}/assets/tldr-book-id.pdf" \
+    "${SITE_HOME}/assets/tldr-book-it.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ja.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ko.pdf" \
+    "${SITE_HOME}/assets/tldr-book-lo.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ml.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ne.pdf" \
+    "${SITE_HOME}/assets/tldr-book-nl.pdf" \
+    "${SITE_HOME}/assets/tldr-book-no.pdf" \
+    "${SITE_HOME}/assets/tldr-book-pl.pdf" \
+    "${SITE_HOME}/assets/tldr-book-pt_BR.pdf" \
+    "${SITE_HOME}/assets/tldr-book-pt_PT.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ro.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ru.pdf" \
+    "${SITE_HOME}/assets/tldr-book-sh.pdf" \
+    "${SITE_HOME}/assets/tldr-book-sr.pdf" \
+    "${SITE_HOME}/assets/tldr-book-sv.pdf" \
+    "${SITE_HOME}/assets/tldr-book-ta.pdf" \
+    "${SITE_HOME}/assets/tldr-book-th.pdf" \
+    "${SITE_HOME}/assets/tldr-book-tr.pdf" \
+    "${SITE_HOME}/assets/tldr-book-uk.pdf" \
+    "${SITE_HOME}/assets/tldr-book-uz.pdf" \
```

Comment:
Same here on trying to use `${SITE_HOME}/assets/tldr-book*.pdf` instead of each individual one. This would avoid the case where accidently including `ja` when it doesn't exist, or forgetting one that was generated above.